### DISCUSSION TITLE
Openapi spec version used for docs

### DISCUSSION THREAD

**Discussion by theankurman** [litestar-org/litestar#1770] [Q&A] [answered]
Hi all,
I was wondering if it's possible to generate openapi v3.0.x schema instead of openapi 3.1.x.

I'm trying to use client generators but couldn't find any that support the 3.1.x version of the schema.

**Comment by Goldziher** [comment, marked as answer]
Hi @a-paudel, we are not planning on doing this. It will require a massive amount of work. We are open to contributions of course - but this is quite an endeavour to do.
